src/styles/custom.css

```diff
@@ -44,6 +44,52 @@ code {
   align-items: center;
 }
 
+.rocket {
+  display: flex;
+  justify-content: space-around;
+  margin: 30px 0;
+}
+
+.rocket-img {
+  max-width: 300px;
+}
+
+.rocket-img img {
+  max-width: 300px;
+}
+
+.rocket-info {
+  width: 100%;
+  padding-left: 30px;
+  display: flex;
+  flex-direction: column;
```

Comment:
## ♻️ Requesting Changes ♻️ 

Great job so far, @alexander16108! Since there's already an index.css file, you don't have to create a new CSS file. To keep things clean, it would be best to transfer this to the existing index.css file. Aside from this, everything else looks good!